locales/en/content.json

```diff
@@ -8,9 +8,12 @@
   "Hed": "Reuters Graphics Interactive",
   "Dek": "The beginning of a beautiful page",
   "Kicker": "Global News",
+  "By": "By",
   "Byline": "**Jane Doe** and **John Doe**",
```

Comment:
Could probably be just one field in the doc for simplicity:

```javascript
{
  "Byline": "By **Jane Doe** and  **John Doe**"
}
```

Reply:
Ohh right of course. I'll change that then merge